(node:94785) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-session-42383',
  containerIp: '10.0.3.81',
  mitmPort: 42383,
  autoAccessFixed: 1
}
2024-11-08 14:30:38.378 - [Info] MITM Version: 2
2024-11-08 14:30:38.380 - [Info] Auto Access Enabled: true
2024-11-08 14:30:38.381 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-08 14:30:38.398 - [Info] SSH man-in-the-middle server for 10.0.3.81 listening on 10.0.3.1:42383
2024-11-08 14:32:28.269 - [Debug] [Connection] Attacker connected: 104.248.129.160 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 14:32:28.866 - [Debug] [Auth] Attacker 104.248.129.160 trying to authenticate with 'password'
2024-11-08 14:32:28.867 - [Debug] [Auto Access] Attacker: 104.248.129.160, Threshold: 1, Attempts: 1
2024-11-08 14:32:28.868 - [Debug] [Auto Access] Compromising the honeypot
2024-11-08 14:32:28.868 - [Debug] [Auto Access] Adding the following credentials: 'jgonzalez:jgonzalez'
2024-11-08 14:32:29.114 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-08 14:32:29.117 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-08 14:32:29.123 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.81
2024-11-08 14:32:29.220 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-08 14:32:29.674 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-08 14:32:30.326 - [Debug] [Connection] Attacker connected: 104.248.129.160 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 14:32:30.925 - [Debug] [Auth] Attacker 104.248.129.160 trying to authenticate with 'password'
2024-11-08 14:32:30.925 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-08 14:32:30.926 - [Debug] [Auth] Attacker: 104.248.129.160 has so far made 1 attempts. Remaining: 5 attempts
2024-11-08 14:32:31.153 - [Debug] [Connection] Attacker closed the connection
2024-11-08 14:32:31.243 - [Debug] [Connection] Attacker connected: 104.248.129.160 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 14:32:31.835 - [Debug] [Auth] Attacker 104.248.129.160 trying to authenticate with 'password'
2024-11-08 14:32:31.835 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-08 14:32:31.836 - [Debug] [Auth] Attacker: 104.248.129.160 has so far made 1 attempts. Remaining: 5 attempts
2024-11-08 14:32:32.057 - [Debug] [Connection] Attacker closed the connection
2024-11-08 14:32:32.244 - [Debug] [Connection] Attacker closed connection
2024-11-08 14:32:32.252 - [Debug] [LXC] Container's OpenSSH server ended connection
2024-11-08 14:32:32.253 - [Debug] [LXC] Container's OpenSSH server closed connection
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
